lib/HeadersGeneration/ModelToHeader.cpp

```diff
@@ -227,10 +215,10 @@ bool dumpModelToHeader(const model::Binary &Model,
           Header << "Analyzing Model function " << FName << "\n";
           serialize(Header, MF);
           Header << "Prototype\n";
-          serialize(Header, *FT);
+          serialize(Header, model::copyTypeDefinition(FT));
```

Comment:
I was going to talk about this in the revng-hour that got postponed.

In short, it's because of the fact that llvm serialization is closely coupled with deserialization. And you can't really deserialize a non-owning polymorphic base.

Reply:
Ah right. Fair enough.